src/avfoundation.cs

```diff
@@ -9160,6 +9160,19 @@ interface AVCapturePhotoSettings : NSCopying
 		[NoWatch, NoTV, NoMac, iOS (12, 0)]
 		[Export ("embedsPortraitEffectsMatteInPhoto")]
 		bool EmbedsPortraitEffectsMatteInPhoto { get; set; }
+
+		[Internal]
+		[NoWatch, NoTV, NoMac, iOS (12, 0)]
+		[Export ("availableRawEmbeddedThumbnailPhotoCodecTypes")]
+		NSString[] _GetAvailableRawEmbeddedThumbnailPhotoCodecTypes { get; }
+
+		[NoWatch, NoTV, NoMac, iOS (12, 0)]
+		[Wrap ("Array.ConvertAll (_GetAvailableRawEmbeddedThumbnailPhotoCodecTypes, s => AVVideoCodecTypeExtensions.GetValue (s))")]
+		AVVideoCodecType[] GetAvailableRawEmbeddedThumbnailPhotoCodecTypes { get; }
```

Comment:
remove `Get` prefix from property

Reply:
@spouliot I'm following the pattern used here: https://github.com/xamarin/xamarin-macios/blob/master/src/avfoundation.cs#L9073

Reply:
bad one, we missed it :(
let's obsolete it and provide a _properly-property-named_ version